crates/sui-core/src/epoch/reconfiguration.rs

```diff
@@ -168,6 +170,8 @@ where
                 "Error when processing advance epoch transaction: {:?}", err
             );
             tokio::time::sleep(WAIT_BETWEEN_EPOCH_TX_QUERY_RETRY).await;
+            cnt += 1;
+            assert!(cnt <= 50);
```

Comment:
Why do we need this?

Reply:
This is temporary for debugging a test failure. Will remove once tests pass